lib/ae_mdw/db/name.ex

```diff
@@ -79,7 +79,7 @@ defmodule AeMdw.Db.Name do
     if Model.name(m_name, :expire) == height do
       owner = Model.name(m_name, :owner)
       cache_through_write(Model.InactiveName, m_name)
-      cache_through_write(Model.InactiveOwnerName, {owner, plain_name})
+      cache_through_write(Model.InactiveNameOwner, {owner, plain_name})
```

Comment:
This fails locally for me as well, because the 2nd parameter of a `:mnesia.write/3` has to be the full record, not just the key of the record

Reply:
probably should leave more time syncing... thank you for finding this!